src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -127,13 +128,21 @@ object EasyStageDataset {
           format = Some(mime),
           sha1 = maybeSha1Map.get.get(bagRelativePath), // first get is checked in advance
           title = title,
-          accessibleTo = FileAccessRights.accessibleTo(rights),
+          accessibleTo = fileAccessRights,
```

Comment:
FileAccessRights.visibleTo always returns ANONYMOUS. I could put it directly here, but it seems better to make these discussions in a central place, i.e. in FileAccessRights.visibleTo.